twill/utils.py

```diff
@@ -128,6 +128,12 @@ def find_link(self, pattern: str) -> Optional[Link]:
             if regex.search(link.text) or regex.search(link.url):
                 return link
         return None
+    
+    def find_links(self, pattern: str) -> Optional[List[Link]]:
```

Comment:
```suggestion
    def find_links(self, pattern: str) -> List[Link]:
```

The Optional is now not needed any more.